### ISSUE TITLE
Error in the PullProducer doc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2174]
|<img alt="misto's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| misto reported|
|-|-|
|Trac ID|trac#2174|
|Type|defect|
|Created|2006-10-18 08:29:30Z|

In the page
http://twistedmatrix.com/projects/core/documentation/howto/producers.html
at the PullProducer paragraph:

errata:The pull producer API is defined by the IPushProducer interface.

corrige: The pull producer API is defined by the IPullProducer interface.

Attachments:

* [twisted.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/0d4/0d4dba0783daabf31a43ce728ae5505e5b301d2a/117d699344a06a0d1955f97b3008aaa77399c4cc.diff) (1422 bytes) - added by misto on 2006-10-18 08:31:34Z - patch

<details><summary>Searchable metadata</summary>

```
trac-id__2174 2174
type__defect defect
reporter__misto misto
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1161160170000000 1161160170000000
changetime__1166862332000000 1166862332000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set status to `closed`|
|-|-|

(In [19036]) Fix typo in "producers" document.

The pull producer API is defined by the IPullProducer interface, not the
IPushProducer interface, obviously.

Author: misto

Reviewer: glyph

Fixes [#2174](https://github.com/twisted/twisted/issues/2174)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
